jdk/test/ProblemList.txt

```diff
@@ -383,3 +383,15 @@ sun/tools/jps/TestJpsJarRelative.java				generic-all
 
 sample/mergesort/MergeSortTest.java				8178912 generic-all
 sample/chatserver/ChatTest.java					8178912 generic-all
+
+############################################################################
+
+# cacerts tests
+
+############################################################################
+security/infra/java/security/cert/CertPathValidator/certification/CAInterop.java#globalsigneccrootcar4  8324583 generic-all
```

Comment:
These 5 test failures from Google root CAs will be resolved with https://git.openjdk.org/jdk/pull/18459

Reply:
Thanks, I've updated the bug number.